bootstrap.sh

```diff
@@ -200,9 +200,9 @@ install-dokku-from-deb-package() {
       OS_ID="bionic"
     fi
   elif [[ "$DOKKU_DISTRO" == "debian" ]]; then
-    OS_IDS=("stretch" "buster")
+    OS_IDS=("stretch" "buster" "bullseye")
```

Comment:
Note that this requires changes to be made to `packagecloud.io`, else should be held back till that is done.

Reply:
Mind also updating this: https://github.com/dokku/dokku/blob/ca680c3aebfd674c0cf52806122708ba3b8ef45d/contrib/release-dokku#L150

Reply:
Done :+1: